lab-anthony/html/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp">
+  <head>
+    <title> cowsay lulwat</title>
+    <link rel="stylesheet" type="text/css" href="bundle.css"/>
+    <script src="bundle.js"></script>
+  </head>
+  <body>
+    <main ng-controller="CowsayController">
+    <form ng-submit="done()">
+      <label for="cowsay-text"> WHAT DOES THE COWSAY? </label>
+      <input
+       name="cowsay-text"
+       type="text"
+       ng-model="cowsayText" />
+      <select ng-model="cowsayFile">
+        <option ng-repeat="cow in cowfiles" value="{{cow}}"> {{cow}} </option>
+      </select>
+      <button type="submit"> DONEZO! </button>
+    </form>
+      <h1>Preview</h1>
+      <pre> {{ say() }} </pre>
```

Comment:
Just like ng-model implements 2 way data binding, 'ng-bind' gives you 1 way data binding. I would use it in place of sharkteeth from now on, as it prevents the 'angular flash'